content/building-blocks/analyze-data/non-parametric-tests/introduction.md

```diff
@@ -0,0 +1,95 @@
+
+## Which test to use?
+Determining which statistical test is most appropriate depends on 3 characteristics of your data:
+
+- Measurement scale
+  1) Nominal / Categorical scale
+      - Categorizes data into distinct groups or categories without any inherent order or numerical value.
+      - Example: Gender, colour
+  2) Ordinal / Ranking scale
+      - Orders or ranks data based on their relative position or preference without making precise statements about the magnitude of the differences between them.      
+      - Example: Grades, Valuations        
+  3) Interval (Cardinal) scale
+      - Categorizes data into distinct groups or levels, and the differences between these categories are meaningful and consistent
+      - Example: Temperature, Calendar dates
+  4) Ratio scale
+      - This is the highest level of measurement scale that possesses all the characteristics of nominal, ordinal, and interval scales, and in addition, it has a true zero point that represents the absence of the measured attribute.
+
+- Type of data
+  - Binary
+  - Discrete
+  - Continuous
+
+- Data structure
+  - One sample
+      - Single observation from a single subject group
+  - Two (or more) related / dependent samples
+      - Multiple observations from a single subject group
+  - Two (or more) independent samples
+      - Multiple observations from several independent subject groups
+
+## Types of non parametric tests
+Now that you know the characteristics of your sample, you can accordingly choose the most appropriate non parametric test
+
+### Binomial Test
+If you are dealing with a `binary data` with a `nominal` measurement scale, binomial test is the most appropriate test. It answers the question: "Is the observed outcome significantly different from what we would expect by a random chance?"; such as: Testing if a coin is fair, testing if a an election outcome favours the Republican or the Democrats, etc.
+
+Check out the building block on the binomial test [# I'll add link here] to learn how to use this test on R and how the formula works mathematically.
+
+### Chi-Squared Goodness of Fit
+Another test you can use with a `binary` and `nominal` data is the Chi-Squared Goodness of Fit. It tests whether a significant difference exists between an observed number of outcomes falling in each category and an expected number based on the null hypothesis (which we call the goodness-of-fit)
+
+Check out the building block on the chi-squared goodness of fit [# I'll add link here] to learn how to use this test on R and how the formula works mathematically.
+
+These 2 tests are useful when you only have 1 sample of data. However, when there are multiple samples of data, it is important to be mindful if those data samples are independent or if they are related to each other. Accordingly you can choose among the following tests:
+
+## Non parametric test for 2 dependent samples
```

Comment:
you write out numbers below 13 (i think that's a convention in English). so: "Non-parametric tests for two dependent samples".

Please explain clearly what these samples are and how they are dependent.